tests/components/hue/test_config_flow.py

```diff
@@ -76,21 +83,104 @@ async def test_flow_works(hass):
     assert result["data"] == {
         "host": "1.2.3.4",
         "username": "home-assistant#test-home",
-        "allow_hue_groups": False,
     }
 
     assert len(mock_bridge.initialize.mock_calls) == 1
 
 
-async def test_flow_no_discovered_bridges(hass, aioclient_mock):
+async def test_manual_flow_works(hass, aioclient_mock):
+    """Test config flow discovers only already configured bridges."""
+    mock_bridge = get_mock_bridge()
+
+    with patch(
+        "homeassistant.components.hue.config_flow.discover_nupnp",
+        return_value=[mock_bridge],
+    ):
+        result = await hass.config_entries.flow.async_init(
+            const.DOMAIN, context={"source": "user"}
+        )
+
+    assert result["type"] == "form"
+    assert result["step_id"] == "init"
+
+    result = await hass.config_entries.flow.async_configure(
+        result["flow_id"], user_input={"id": "manual"}
+    )
+
+    assert result["type"] == "form"
+    assert result["step_id"] == "manual"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5795327)